lib/hawkular/inventory/entities_v4.rb

```diff
@@ -0,0 +1,95 @@
+# It contains class definitions that are used by the inventory REST client
+module Hawkular::InventoryV4
```

Comment:
Perhaps we could baptise the new inventory as `Hawkular::Registry`. I think _registry_ is more appropiate :)